src/ontology/cl-edit.owl

```diff
@@ -27904,6 +27908,21 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31881018"^^xsd:string) A
 AnnotationAssertion(rdfs:label obo:CL_4023164 "globular bushy cell")
 SubClassOf(obo:CL_4023164 obo:CL_4023162)
 
+# Class: obo:CL_4023168 (somatosensory neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22865660"^^xsd:string) obo:IAO_0000115 obo:CL_4023168 "A neuron that is part of the somatic sensory system. Somatosensory neurons innervate the skin or integument to detect different types of thermal, chemical, and mechanical touch stimuli.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023168 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023168 "somatosensory neuron")
+EquivalentClasses(obo:CL_4023168 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003942)))
+
+# Class: obo:CL_4023169 (trigeminal neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22190641"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Trigeminal_nerve&oldid=1099897926"^^xsd:string) obo:IAO_0000115 obo:CL_4023169 "A sensory neuron that is responsible for sensation in the face and motor functions such as biting and chewing. Trigeminal neurons extend a single axon shaft along the lateral white matter of the hindbrain and spinal cord. Numerous medially projecting branches are later formed and innervate a series of target nuclei along the anteroposterior axis of the hindbrain and spinal cord. The highly branched axons innervate the integument of the head.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023169 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023169 "trigeminal neuron")
+SubClassOf(obo:CL_4023169 obo:CL_4023168)
+SubClassOf(obo:CL_4023169 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001645))
```

Comment:
@dosumis - I'm not 100% sure about this, do we have conventions on how we related neurons to nerves?

Reply:
btws for ease - that is `'has soma location' some 'trigeminal nerve'`

Reply:
How about `'neuron' 'part of' some 'trigeminal nerve'`? 

Definition starts by stating "A sensory neuron..." and then mentions it is capable of motor function. Depending on branch, it could be sensory or sensorimotor (https://www.ncbi.nlm.nih.gov/books/NBK539823/). Don't see motor function reflected in logical axioms, only somatosensory. 

Definition gloss: "Numerous medially projecting branches are later formed..." This seems more applicable to nerve than neurone.

Re: soma location, would 'trigeminal ganglion' be more appropriate? I'm unsure if that is true for all branches though.

Reply:
> How about 'neuron' 'part of' some 'trigeminal nerve'

I'm not super comfortable with 'part of' cause with neurons, we are trying to consistently use soma location, but also, you comment on the bottom has a good point on that. 

> Definition starts by stating "A sensory neuron..." and then mentions it is capable of motor function. 

Good catch - I think a solution might be to have trigeminal sensory neuron & trigeminal motor neuron under trigeminal neuron

> Re: soma location, would 'trigeminal ganglion' be more appropriate? I'm unsure if that is true for all branches though.

Yeah ganglion would defs be more appropriate. Looking at other similar types, might be wise to remove that axiom for now.